requirements.txt

```diff
@@ -1,11 +1,12 @@
-django>=2.1,<2.2
-https://github.com/opencivicdata/python-opencivicdata-django/archive/2.5.0.zip
-pupa==0.9.0
+django>=2.2,<2.3
+opencivicdata>=3.1.0
+pupa==0.10.1
```

Comment:
Includes post-save signal after related object import and fix for default to all scrapers if none specificed: https://github.com/opencivicdata/pupa/commit/158378e19bcc322796aa4fb766784cbd4fd08413.